internal/provider/resource_tfe_workspace.go

```diff
@@ -722,6 +757,15 @@ func resourceTFEWorkspaceUpdate(d *schema.ResourceData, meta interface{}) error
 			}
 		}
 
+		if tagBindings, ok := d.Get("tag_bindings").(map[string]interface{}); ok {
```

Comment:
Removing tag_bindings config or setting it to `{}` did not unset the tags.

Reply:
Investigating a bit further the culprit is actually the `omitempty` in go-tfe, which omits empty slices from being sent over the wire. This would require a change to add a `RemoveTagBindings()` receiver for projects/workspaces that explicitly sends an empty array to remove all bindings.